kong.conf.default

```diff
@@ -447,6 +447,22 @@
                                  # See the `lua_ssl_trusted_certificate`
                                  # setting to specify a certificate authority.
 
+#pg_max_concurrent_queries = 0   # Sets the maximum number of concurrent queries
+                                 # that can be executing at any given time. This
+                                 # limit is enforced per worker process; the
+                                 # total number of concurrent queries for this node
+                                 # will be will be `pg_max_concurrent_queries * nginx_worker_processes`.
```

Comment:
It'd be nice to stick to the 80 chars limit in this file

Reply:
I want to say I found a few existing exceptions and based my thoughts on this. Let me 3x check and adjust if I'm off base

Reply:
The only exception we let slip through has been for links (or _should_ still be, if other oversights slipped through...)

Reply:
Yep. In this case moving the string "\`pg_max_concurrent_queries * nginx_worker_processes\`." still results in exceeding the limit based on how far we're indented. I (or the merger) can split this into multiple lines, but that feels a little awkward.